authortechnical.Rmd

```diff
@@ -132,7 +132,7 @@ Refer [to this tutorial for details on what should go in alternative text](https
 - Control image **placement** with `class`  
   `{{< figure src = "image-name.png" alt = "informative description" class = "center" >}}`
     - `pull-left` - Left-align the picture and wrap text around it
-    - `center` - Centre the picture (no text wrapping)
+    - `center` - Center the picture (no text wrapping)
```

Comment:
lol

Reply:
(I'm sorry for the typo, of course!!)

Reply:
Oh that's my typo :grin: I'm Canadian, so I have to force myself to use American spelling with rOpenSci (it hurts!)